cpp/oneapi/dal/algo/svm/detail/infer_ops.hpp

```diff
@@ -58,10 +58,10 @@ struct infer_ops {
             throw invalid_argument(
                 "Input model support_vectors row_count should be equal to input model support_vectors_count");
         }
-        if (input.get_model().get_coefficients().get_row_count() !=
+        if (input.get_model().get_coeffs().get_row_count() !=
             input.get_model().get_support_vector_count()) {
             throw invalid_argument(
-                "Input model coefficients row_count should be equal to input model support_vectors_count");
+                "Input model coeffs row_count should be equal to input model support_vectors_count");
```

Comment:
```suggestion
                "Row count of coefficients should be equal to support vectors count in the input model");
```

Reply:
We had a rule in this order to draw up exceptions. @mchernov-intel Correct me if I'm wrong.